microsite/data/plugins/deploy.yaml

```diff
@@ -0,0 +1,13 @@
+---
+title: Deploy
```

Comment:
Name is too generic - could be `Digital.ai Deploy` for example? Same with the naming of this file and the image below.

Reply:
Hi @freben , 
Thanks for reviewing the PR.
We have updated the name.